CHANGELOG.md

```diff
@@ -14,6 +14,8 @@
 
 - [BUGFIX] Fix race condition that may occur and result in a panic when
   initializing scraping service cluster. (@rfratto)
+  
+- [BUGFIX] Upgrade dependecy that caused builds in 32-bit architectures to fail. (@mapno)
```

Comment:
You only need to add a changelog entry if it crosses release boundaries :) Since this is only an issue in main we can take this out